### DISCUSSION TITLE
Growatt - MOD10KTL3-XH

### DISCUSSION THREAD

**Discussion by rodeoric** [wills106/homeassistant-solax-modbus#1185] [Q&A]
Hello, 
i have the following problem. 
I am running HA in a Proxmox Veeam. 
SolaX is running fine since about 7-8 months.

Now, when i do a full reboot (means settings -> system > right top corner > home assistant reboot) the full solax integration is not working anymore. Modbus seems to get no connection anymore. I always do a snapshot of my VM before i do anything on HA. When i get the problem after a full reboot, i couldnt fix it at all. I tried anything, deactivate the integration, reboot (normal), restart addon, i reconfigured the integration, i unplugged and plugged in the usb again, nothing was working to get it fixed, i had to reload the snapshot and all was working fine again. 

Anyone have any idea or have the same problem or should i open a bug report ?

**Comment by PSMGoossens** [comment]
@rodeoric 
Could you provide us with any debug log? I'm using the same inverter (less power) with even the latest firmware version from growatt, but I don't get any problem.
